docs_list.md

```diff
@@ -0,0 +1,143 @@
+# Docs list
+
+A working list of docs with links to the published documentation, the repo or repos, and the intended purpose of the technical content.
+
+## Cosmos Hub
+
+- [Cosmos Hub](https://hub.cosmos.network/main/hub-overview/overview.html)
+
+- <https://github.com/cosmos/gaia/tree/master/docs>
+
+- Using the Cosmos SDK application for the Cosmos Hub.
+
+  <details><summary>Top-level contents</summary>
+
+  - Cosmos Hub
+
+  - Delegators
+
+  - Gaia Tutorials
+
+  - Validators
+
+  </details>
+
+## Cosmos SDK Documentation
+
+- [Cosmos SDK Documentation](https://docs.cosmos.network/) _why 2 repos?_
```

Comment:
```suggestion
- [docs.cosmos.network](https://docs.cosmos.network/)
```

This is the docs for the cosmos sdk _framework_. The Cosmos Hub (gaia) is one instance of an application using the Cosmos SDK (also the most important application using it because it manages ATOMs which pay all our salaries 🤑)

The Cosmos SDK in general is meant to be used by many different companies / networks